java/client/src/main/java/glide/api/commands/ServerManagementCommands.java

```diff
@@ -13,8 +13,8 @@
 public interface ServerManagementCommands {
 
     /**
-     * Get information and statistics about the Redis server. No argument is provided, so the {@link
-     * Section#DEFAULT} option is assumed.
+     * Get information and statistics about the Redis server. If no argument is provided, so the
```

Comment:
No need to include "If" since there is no way to add an argument on this command. 
Maybe: 
```
Get information and statistics about the Redis server using the {@link Section#DEFAULT} option. 
```

Reply:
Thanks you, updated in 10277d3f